app/models/pieces.py

```diff
@@ -18,6 +19,37 @@ class Pawn(Piece):
     def validate_move(self, board: np.array, move: PieceMove):
         super().validate_move(board, move)
 
+        # Get the color of the piece at the starting position
+        color = self.get_color(board, move.from_pos)
+
+        # 1 for white (moves up), -1 for black (moves down)
+        direction = 1 if color == "white" else -1
```

Comment:
We could use a Model class maybe for "direction" and "color". But we could do it with a refactor later.
Let's just proceed with this step now.